lua/lvim/core/terminal.lua

```diff
@@ -78,6 +78,8 @@ local function get_dynamic_terminal_size(direction, size)
 end
 
 M.init = function()
+  lvim.builtin.terminal.shell = lvim.builtin.terminal.shell or vim.o.shell
+
```

Comment:
```suggestion
```

vim.o.shell is toggleterm's default so it will use it if we pass nil

Reply:
<img width="889" alt="Screenshot 2023-02-18 at 21 18 20" src="https://user-images.githubusercontent.com/42694704/219870746-91ce5a4c-cbdf-42b1-97ab-5a0d1d0b7fdb.png">

There was an error because of this line 
https://github.com/LunarVim/LunarVim/blob/f43cc8e20b8443f052c8bd1dd74066eec514587b/lua/lvim/core/terminal.lua#L109

Reply:
In that case I'd change line 85 https://github.com/LunarVim/LunarVim/blob/f43cc8e20b8443f052c8bd1dd74066eec514587b/lua/lvim/core/terminal.lua#L85
to
`cmd = exec[1] or lvim.builtin.terminal.shell or vim.o.shell,`

Reply:
You are right, that's more concise